packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -125,22 +125,16 @@ export class VaCheckbox {
 
   private fireAnalyticsEvent = () => {
     // Either the description prop or the text content of the description slots
-    const description =
-      this.description ||
-      [
-        // This won't work in IE
-        ...(
-          this.el.shadowRoot.querySelector(
-            'slot[name="description"]',
-          ) as HTMLSlotElement
-        )?.assignedNodes(),
-        // For IE
-        ...Array.from(
-          this.el.shadowRoot.querySelectorAll('[slot="description"]'),
-        ),
-      ]
-        .map(n => n.textContent)
-        .join(' ');
+    let description = this.description || '';
```

Comment:
I am making this falsy with `''` so that the `description` key still gets sent with the analytics payload. It'll just be blank.